cmd/helm/init.go

```diff
@@ -166,6 +173,14 @@ func (i *initCmd) tlsOptions() error {
 				return errors.New("missing required TLS CA file")
 			}
 		}
+
+		// FIXME: refactor this all to pkg/helm/environment
```

Comment:
Does the FIXME still need to be here?

Reply:
Yeah. I'd like to rip this all out in Helm 3 and replace it with the same flags used elsewhere. I'll update the comment to reflect that there's nothing immediately actionable here